src/services/compare-with-azion/get-test.js

```diff
@@ -0,0 +1,66 @@
+import { AxiosHttpClientAdapter } from '../axios/AxiosHttpClientAdapter'
+import makeCompareWithAzionApi from '../axios/makeCompareWithAzionApi'
+import { makeWebpagetestBaseUrl } from './make-base-url'
+
+export const getTestById = async (id, clientId = undefined) => {
```

Comment:
Por default o clientId é undefined quando não passado.
```suggestion
export const getTestById = async (id, clientId) => {
```

Reply:
Feito, validação client_id quando não existente, nem deixa iniciar essas chamadas!

Mudei para ficar no padrão, porém a integridade de parâmetros mesmo não passado deve ser integro e saber dar alguma responta em todas camadas no lugar  de se invalido quebrar.